enterprise_access/apps/bffs/api.py

```diff
@@ -260,9 +274,10 @@ def _determine_enterprise_customer_for_display(
     Returns:
         The enterprise customer user for display.
     """
+
     if not enterprise_customer_slug and not enterprise_customer_uuid:
         # No enterprise customer specified in the request, so return the active enterprise customer
-        return active_enterprise_customer
+        return active_enterprise_customer, False
 
     # If the requested enterprise does not match the active enterprise customer user's slug/uuid
     # and there is a linked enterprise customer user for the requested enterprise, return the
```

Comment:
nit: this docstring could be updated to indicate updating the `active` ECU, per the `True` in the returned tuple